docs/source/extend_kedro/plugins.md

```diff
@@ -6,6 +6,24 @@ Kedro plugins allow you to create new features for Kedro and inject additional c
 
 Kedro's extension mechanism is built on [`pluggy`](https://pluggy.readthedocs.io/), a solid plugin management library that was created for the [pytest](https://docs.pytest.org/) ecosystem. `pluggy` relies on [entry points](https://packaging.python.org/en/latest/specifications/entry-points/), a Python mechanism for packages to provide components that can be discovered by other packages using [`importlib.metadata`](https://docs.python.org/3/library/importlib.metadata.html#entry-points).
 
+## Kedro-Telemetry
+
+[The Kedro-Telemetry plugin](https://github.com/kedro-org/kedro-plugins/tree/main/kedro-telemetry) is an opt-in feature designed to gather anonymized and aggregated usage analytics, ensuring no personal data is collected. Its aim is to understand how Kedro's features are utilized to inform future product development and enhance the user experience. This telemetry data, approved under the [Telemetry Data Collection and Usage Policy](https://lfprojects.org/policies/telemetry-data-policy/) of LF Projects, LLC, ensures privacy while providing valuable insights to [Kedro Project maintainers](https://docs.kedro.org/en/stable/contribution/technical_steering_committee.html).
+
+### Collected Data Fields:
```

Comment:
```suggestion
### Collected data fields:
```

Reply:
I'd like to see all sentence case across the page but again, please check if this is OK with whoever supplied the text. Or ignore this suggestion and stick with Camel Case.